tools/tune/eq/mls_freq_resp.m

```diff
@@ -118,23 +118,28 @@
 		remote_capture(recfn, rec_cfg, tcap);
 		pause(1);
 		remote_play(mixfn, play_cfg);
-		pause(3);
+		pause(5);
 		r = get_recording(recfn, rec_cfg);
 	end
-	[d, nt] = find_test_signal(r(:,1), fnd);
+    [d, nt] = find_test_signal(r(:,1), fnd);
+    figure;
+    sr = size(r);
+    ts = (0:sr(1)-1)/fs;
+    plot(ts, r(:,1));
+    grid on;
+    xlabel('Time (s)');
+    ylabel('Sample value');
 	if isempty(d)
-		figure;
-		sr = size(r);
-		ts = (0:sr(1)-1)/fs;
-		plot(ts, r(:,1));
-		grid on;
-		xlabel('Time (s)');
-		ylabel('Sample value');
 		title('Captured audio test waveform');
 		fprintf(1, 'Error: check the plot for skew in capture/playback.\n');
 		f = [];
 		m_db = [];
 		return
+    else
+        si = d:d + nt;
```

Comment:
just d isn't sufficient ? d:d is d

Reply:
Operator : has lower precedence than +, this could be d:(d + nt) to be more clear. But this worked, otherwise there would be no green plot.

Reply:
Found it https://se.mathworks.com/help/matlab/matlab_prog/operator-precedence.html